modules/coo-distributed-tracing-ui-plugin-using.adoc

```diff
@@ -16,12 +16,21 @@
 
 .Procedure
 
-. In the {ocp-product-title} web console, click **Observe** → **Traces**.
-. Select a `TempoStack` or `TempoMonolithic` multi-tenant instance and set a time range and query for the traces to be loaded.
+. In the Administrator perspective of the {ocp-product-title} web console, click *Observe* → *Traces*.
```

Comment:
Since OCP 4.19 there is no dedicated Administrator perspective anymore: https://developers.redhat.com/articles/2025/06/26/openshift-419-brings-unified-console-developers-and-admins